e2e_samples/parking_sensors/adf/factory/mdwdops-adf-dev-16.json

```diff
@@ -0,0 +1,9 @@
+{
+	"name": "mdwdops-adf-dev-16",
+	"location": "australiaeast",
```

Comment:
make it "westus"?

Reply:
This is same as the other linkedservices and gets overriden in the deployment. I've added a note in the readme to clarify :)